packages/react-icons/src/utils/SvgIcon.scss

```diff
@@ -0,0 +1,13 @@
+.root {
+  display: inline-block;
+  vertical-align: middle;
+  speak: none;
+  width: 1em;
+  height: 1em;
+}
+
+.svg {
+  height: 100%;
```

Comment:
is `auto` the default value for `width`? I think so, not sure tho.

Reply:
`width` default value is `auto`: https://www.w3schools.com/cssref/pr_dim_width.asp